src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import React, { useState } from 'react';
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelect }) => {
+  const [activeTabId, setActiveTabId] = useState(tabs[0].id);
```

Comment:
Here your code still hasn't passed the requirements:
![image](https://github.com/mate-academy/react_tabs-js/assets/25587299/f9cb0878-5fd4-4c6a-83c2-2c72ec73d8bd)

Reply:
<img width="781" alt="Screenshot 2023-10-30 at 15 26 37" src="https://github.com/mate-academy/react_tabs-js/assets/135232001/1ff7e486-9974-490e-a00b-79374c4b9294">
Not fixed from the previous review